t/nextjs_react_template/contents/src/app/auth.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { useUser } from '@auth0/nextjs-auth0/client';
+import { Button } from '@mui/material';
+import { AUTH_LOADING_TEXT, AUTH_LOGIN_TEXT, AUTH_LOGOUT_TEXT } from './constants';
+
+export default function Auth() {
+  const { user, error, isLoading } = useUser();
+
+  if (isLoading) return <div>{AUTH_LOADING_TEXT}</div>;
+  if (error) return <div>{error.message}</div>;
+
+  if (user) {
+    return (
+      <div>
+        Welcome {user.email}!
```

Comment:
The PR generally lgtm, but for layout, we recommend use MUI everywhere for consistency. E.g. Typography here.